### PR TITLE
feat(ckan): Add ckan values variable

### PR DESCRIPTION
**Pull request by SyeKlu**
by using the variable for the plugins, the plugins can now be customized by the user. it also allows the use of the DCAT-AP.de plugin from a ckan container.

### CODE REVIEW

File: charts/ckan/values.schema.json [hunk 1]
```diff
@@ -63,6 +63,10 @@
           "type": "string",
           "description": "This sets the name of the site, as displayed in the CKAN web interface."
         },
+        "plugins":{
+          "type": "string",
```
Comment (cwrau) [inline charts/ckan/values.schema.json:67]: This should rather be an array of strings to be more user friendly

File: charts/ckan/templates/ckan/deployment.yaml [hunk 2]
```diff
@@ -138,7 +138,7 @@ spec:
             - name: CKAN__DATAPUSHER__CALLBACK_URL_BASE
               value: "http://{{ printf "%s-%s" (include "common.names.fullname" $) "ckan" }}:{{ include "ckan.ckan.service.port" $ }}/"
             - name: CKAN__PLUGINS
-              value: "envvars image_view text_view recline_view datastore datapusher"
+              value: {{ .Values.ckan.plugins | quote }}
```
Comment (cwrau) [inline charts/ckan/templates/ckan/deployment.yaml:141]:
```suggestion
              value: {{ .Values.ckan.plugins | join " " | quote }}
```

File: charts/ckan/values.yaml [hunk 3]
```diff
@@ -26,6 +26,7 @@ defaultSecurityContext: &defaultSecurityContext
   runAsUser: 1000
 
 ckan:
+  plugins: "envvars image_view text_view recline_view datastore datapusher"
```
Comment (cwrau) [inline charts/ckan/values.yaml:29]:
```suggestion
  plugins: 
    - envvars
    - image_view
    - text_view
    - recline_view
    - datastore
    - datapusher
```

**Review by cwrau — CHANGES_REQUESTED** [review]

File: charts/ckan/values.schema.json (hunk 1 above)
Comment (SyeKlu) [reply charts/ckan/values.schema.json]: you are right it is the nicer version. i have adapted it

File: charts/ckan/values.schema.json [hunk 4]
```diff
@@ -63,6 +63,16 @@
           "type": "string",
           "description": "This sets the name of the site, as displayed in the CKAN web interface."
         },
+        "plugins":{
+          "type": "array",
+          "items": {
+            "items": {
+              "type": "string"
+            },
+            "additionalProperties": false
```
Comment (cwrau) [inline charts/ckan/values.schema.json:72]: This is not needed

```suggestion
            }
```

**Review by cwrau — CHANGES_REQUESTED** [review]

File: charts/ckan/values.schema.json [hunk 5]
```diff
@@ -63,6 +63,16 @@
           "type": "string",
           "description": "This sets the name of the site, as displayed in the CKAN web interface."
         },
+        "plugins":{
+          "type": "array",
+          "items": {
+            "items": {
+              "type": "string",
+              "additionalProperties": false
```
Comment (cwrau) [inline charts/ckan/values.schema.json:71]:
```suggestion
              "type": "string"
```

**Review by cwrau — APPROVED** [review]

**Review by cwrau — APPROVED** [review]

**Review by cwrau — APPROVED** [review]

**Review by tasches — APPROVED** [review]

**Pull request merged**